client/src/components/Profile/UserProfile/UserProfile.js

```diff
@@ -226,7 +226,7 @@ class UserProfile extends Component {
   render() {
     console.log(this.state);
     // console.log(this.props.location.pathname);
-    // console.log(this.props.match.params); 
+    // console.log(this.props.match.params);
```

Comment:
@simonnguyen3054 What was the change that you made here?

Reply:
ohh there shouldn't be any change, I was just looking at your state refactoring to services. Maybe I added a space accidently